tools/mappingsGenerator.ts

```diff
@@ -0,0 +1,42 @@
+const fg = require('fast-glob');
+const fs = require('fs');
+const yaml = require('yaml');
+
+type Path = string;
+type ThemeName = string;
+type TokenKey = string;
+type TokenMapping = string;
+
+console.log(`> theme configs loading...`)
+
+const configs: Path[] = fg.sync('src/themes/*.theme.json');
+
+console.log(`> found ${configs.length} theme paths: \n\t${configs.join('\n\t')}\n`)
+
+const mappers: Record<ThemeName, Path[]> = configs.reduce((acc, path) => {
```

Comment:
Тут кстати можно не писать два раза типы:

```suggestion
const mappers = configs.reduce<Record<ThemeName, Path[]>>((acc, path) => {...}, {})
```